lib/src/assets/icons.dart

```diff
@@ -1,2192 +1,2177 @@
 // ignore_for_file: public_member_api_docs, constant_identifier_names
 import 'package:flutter/material.dart';
 
-const zetaIconsVersion = '0.5.1';
+// This file is automatically generated by the zeta-icons respository
+// DO NOT MODIFY
+
+const zetaIconsVersion = '0.5.2';
 
 /// Zeta Icons. Full list of icons can be found at [Zeta Icons](https://zeta-icons.web.app/).
-class ZetaIcons {
-  ZetaIcons._();
-  static const String _familyRound = 'zeta-icons-round';
-  static const String _familySharp = 'zeta-icons-sharp';
-  static const String _package = 'zeta_flutter';
+abstract class ZetaIcons {
```

Comment:
i think this could be `abstract final class`. 
I don't know if that is actually beneficial

Reply:
looks like it just prevents inheritance https://dart.dev/language/class-modifiers which doesn't really help us that much